src/apps/activity_assignments/service.py

```diff
@@ -234,6 +239,54 @@ async def get_all(self, applet_id: uuid.UUID, query_params: QueryParams) -> list
             for schema in schemas
         ]
 
+    async def get_all_by_respondent(
+        self, applet_id: uuid.UUID, respondent_subject_id: uuid.UUID
+    ) -> list[ActivityAssignmentWithSubject]:
+        schemas = await ActivityAssigmentCRUD(self.session).get_by_applet_and_respondent(
```

Comment:
```suggestion
        assignments = await ActivityAssigmentCRUD(self.session).get_by_applet_and_respondent(
```

Reply:
Same here, not a big deal if we leave this in

Reply:
done